management/WFCORE-6830_Management_Resource_Constraints.adoc

```diff
@@ -0,0 +1,251 @@
+application server, this feature request is not intended to revisit the existing behaviour which is already available at
+all stability levels.
+
+=== Future Work
+
+The only possible future work could be to promote one or all of these new attributes to a higher stability level.
+
+A change such as this should likely be constrained to:
+
+ * Removing the stability level qualifier from the attribute definitions allowing them to be used at the default stability level.
+ * Moving the XML schema definitions and parsing to an appropriate schema at the default stability level.
+ * The community documentation can be simplified to remove references to the stability level
+ `Community` once supported at all stability levels.
+
+== Backwards Compatibility
+
+In considering backwards compatibility if an attribute is defined on the HTTP management interface AND the
+corresponding system property is set then the system property should take precedence.
+
+For each of the system properties if defined and if the attribute is supported at the current stability level  a message in the
+following format will be logged to inform the administrator that they should move to using the new attribute definitions.
+
+[source]
+----
+14:58:13,878 INFO  [org.jboss.as.controller.management-deprecated] (Controller Boot Thread) WFLYCTL0515: The system
+property 'org.wildfly.management.backlog' is deprecated and may be removed in a future version, attribute 'backlog' on
+resource '/core-service=management/management-interface=http-interface' should be used instead.
+----
+
+=== Default Configuration
+
+The new attributes will not be present in the default configuration, instead the default values will remain in use.
+
+=== Importing Existing Configuration
+
+As the attributes are optional and defined with default values the changes here will not impact the ability to import
+and existing configuration.
+
+=== Deployments
+
+N/A
+
+=== Interoperability
+
+N/A
+
+== Admin Clients
+
+The JBoss CLI utilises low level management operations when updating the http-management-interface, these dynamically
+take into account the results of calling :read-resource-description so these new attributes will be automatically
+supported by the CLI when the server is running at the `Community` stability level or lower.
+
+The admin console does contain a screen dedicated to the management of the HTTP management interface available at
+http://localhost:9990/console/index.html#standalone-server[HTTP Management Interface].  This screen is dynamically rendered
+based on the attributes defined on the HTTP management interface resource, this means that when the server is running at
+a stability level that supports these new attributes the admin console will automatically support them - in other cases they
+will not be present in the view.
+
+No follow up tasks are required for either admin client.
```

Comment:
OT: We should make this easily cut-and-pasted content; perhaps in a more fleshed out https://github.com/wildfly/wildfly-proposals/blob/main/FEATURE_PROCESS.adoc